storage/storage.go

```diff
@@ -81,6 +81,10 @@ const (
 	// ScopeReadWrite grants permissions to manage your
 	// data in Google Cloud Storage.
 	ScopeReadWrite = raw.DevstorageReadWriteScope
+
+	// defaultConnPoolSize is the default number of connections
+	// to initialize in the GAPIC gRPC connection pool.
```

Comment:
Maybe for later but do we want to add a comment about how a larger pool may be required for high-throughput operations?

Reply:
I will add something here, but the proper documentation I think should be written when we take this public.

Reply:
Added a bit.